handler/handler.go

```diff
@@ -27,8 +29,15 @@ func CreateNatsQueue(address string, port int) (*NatsQueue, error) {
 	clientID := "faas-publisher-" + val
 	clusterID := "faas-cluster"
 
-	nc, err := stan.Connect(clusterID, clientID, stan.NatsURL(natsURL))
-	nc.NatsConn().SetReconnectHandler(queue1.reconnectClient(clientID, clusterID, natsURL))
+	natsConn, err = nats.Connect(natsURL, nats.ReconnectHandler(queue1.reconnectClient(clientID, clusterID, natsURL)))
+	if err != nil {
+		return nil, err
+	}
+
+	nc, err := stan.Connect(clusterID, clientID, stan.NatsConn(natsConn))
+	if err != nil {
+		return nil, err
+	}
 	queue1.nc = nc
```

Comment:
You should store in queue1 also the nats connection in order to close (if the code is ever closing on exit, etc..)

Reply:
Note: I don't see anywhere a Close() or the like in the handler, but still, keep track of it in case this is changed later.

Reply:
@vosmith have you addressed this?

Reply:
Hey @alexellis , there is no place to close this as the connection should stay open as long as the webserver is up and running.  AFAIK there isn't a way to handle a graceful shutdown.